static/chat.delta.desktop.appdata.xml

```diff
@@ -1,173 +1,160 @@
 <!--<?xml version="1.0" encoding="UTF-8"?>-->
 <component type="desktop">
   <id>chat.delta.desktop</id>
-  <metadata_license>CC0</metadata_license>
+  <launchable type="desktop-id">chat.delta.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-3.0-or-later</project_license>
   <name>Delta Chat</name>
   <summary>Delta Chat email-based messenger</summary>
-  
+
   <description>
-    <p>The messenger with the broadest audience in the world.
-Free, independent, email compatible.
-    </p>
+    <p>The messenger with the broadest audience in the world. Free, independent, email compatible.</p>
   </description>
-  
-​  <launchable type="desktop-id">chat.delta.desktop</launchable>
 
   <screenshots>
     <screenshot type="default">
       <image>https://github.com/deltachat/deltachat-desktop/raw/master/screenshot.png</image>
     </screenshot>
   </screenshots>
 
-
- <url type="homepage">https://delta.chat/</url>
+  <url type="homepage">https://delta.chat/</url>
+  <url type="bugtracker">https://github.com/deltachat/deltachat-desktop/issues</url>
+  <url type="faq">https://delta.chat/en/help</url>
+  <url type="donation">https://delta.chat/en/contribute</url>
+  <url type="translate">https://www.transifex.com/delta-chat/public/</url>
 
   <releases>
     <release version="v0.98.2" date="2019-01-11">
-        <description>
-        <p>
-        Changed
-
-            Tweak search button (@jikstra)
-            Convert src/config.js to src/applications-constants.js with a functional API (@ralphtheninja)
-            Improve css building for conversations stylesheets (@jikstra)
-            Update install instructions in README (@jikstra, @ralphtheninja)
-            Restyle create chat buttons (@jikstra)
-            Update outdated watch script (@ralphtheninja)
-            Hide known accounts section when it's empty (@Simon-Laux)
-            Set minimum window height to 450px (@jikstra)
-            Upgrade deltachat-node to ^0.36.0 for Mac OS prebuilt binaries (@ralphtheninja)
-            Make unit tests less spammy (@ralphtheninja)
-            Simplify state load (@ralphtheninja)
-            Make converting translations less spammy (@ralphtheninja)
-
-        Added
-
-            Add rc module for configuration (@ralphtheninja)
-            Add logging functionality (@Simon-Laux)
-            Add hallmark module for markdown linting (@ralphtheninja)
-
-        Removed
-
-            Remove bin/clean.js (@ralphtheninja)
-            Clean up unused configuration code (@ralphtheninja)
-
-        Fixed
-
-            Fix non verified contacts in verified groups (@ralphtheninja)
-            Fix escaped characters in translations (@ralphtheninja)
-            Adjust login form so it's not hidden below navigation bar (@jikstra)
-            Fix broken rimraf dependency (@jikstra)
-        </p>
-        </description>    
```

Comment:
I'm unsure if it makes sense to have the changelog in appdata.xml as long as we don't display it somewhere. And i think i never saw it in the desktop yet, maybe it's somewhere hidden ^^ @har9862 do you know more about this and what it's goo for?